sn_networking/src/lib.rs

```diff
@@ -356,9 +373,16 @@ impl SwarmDriver {
             .multiplex(libp2p::yamux::Config::default())
             .boxed();
 
+        #[cfg(feature = "quic")]
+        let quic_transport = libp2p_quic::tokio::Transport::new(quic::Config::new(&keypair));
+        #[cfg(feature = "quic")]
+        let mut transport = quic_transport
+            .map(|(peer_id, muxer), _| (peer_id, StreamMuxerBox::new(muxer)))
+            .boxed();
```

Comment:
Just a nit pick, we can probably combine these for better readability?
```suggestion
        #[cfg(feature = "quic")]
        let mut transport = libp2p_quic::tokio::Transport::new(quic::Config::new(&keypair))
            .map(|(peer_id, muxer), _| (peer_id, StreamMuxerBox::new(muxer)))
            .boxed();
```